modules/core/shared/src/main/scala/gem/enum/PixelScale.scala

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.util.Enumerated
+
+/**
+ * Enumerated type for Pixel Scale.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class PixelScale(
```

Comment:
You asked about this and I mentioned that other instruments have the same concept.  They won't match the GNIRS values though so this should probably be `GnirsPixelScale`.

Reply:
:) I actually had it first as `GnirsPixelScale` but changed it when I had to add it to `EnumDef`. There is no instrument specific code in `EnumDef`, is it fine to have `GnirsPixelScale` in `EnumDef`? Where should I place it otherwise?

Reply:
Hmm, unfortunately I don't know of another alternative.

Reply:
In `EnumDef` is fine with me. We can split it up at some point if it gets to be too much of a junk drawer.